File: PlayMinds-Back/src/main/java/com/coderhouse/pmb/Entitys/Notification.java
package com.coderhouse.pmb.Entitys;

import jakarta.persistence.*;
import lombok.Data;

import java.io.Serializable;

@Data
@Entity
@Table(name = "notification")
public class Notification implements Serializable {
    @Id
    @Column(name = "id_notification")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String message;
    private boolean viewed;
    private String redirect;

    @Column(name = "FK_email")
    private String user;
}

File: play-minds/src/models/Entitys/Notification.ts
export class Notification {
  id?: number;
  message: string;
  viewed: boolean;
  redirect: string;
  user: string;

  constructor(
    id: number,
    message: string,
    viewed: boolean,
    redirect: string,
    user: string
  ) {
    this.id = id;
    this.message = message;
    this.viewed = viewed;
    this.redirect = redirect;
    this.user = user;
  }
}
